terraform/data_aoc_msk/vars.tf

```diff
@@ -0,0 +1,20 @@
+// Prefix used in the cluster name
+variable "cluster_name_prefix" {
+    default = "AOCMSKCluster"
+}
+
+// The Kafka Cluster version
+variable "cluster_version" {
+    default = "2.8.1"
+}
+
+// Used to make sure that we don't have topic collisions. We cannot generate infinite amount of topics
+// but we also don't want tests to collide.
+variable "dedup_topic" {
+    default = ""
+}
+
+// Testcase name as passed to each main module (ec2, ecs or eks). E.g.: -var="testcase=../testcases/statsd"
+variable "testcase" {
+    default = ""
+}
```

Comment:
Is this not available already in `common`?

Reply:
I think I see now, we have to pass it into the `data_aoc_msk` module because this module does not import `common`.

Reply:
Correct. my idea was to make this module totally self contained. It is basically used to enforce the naming conventions for looking up for clusters.